src/factorial.js

```diff
@@ -0,0 +1,13 @@
+export default function factorial(num){
+  if(num > 0){
```

Comment:
Check spacing on this file since you have changed it in the `setIntersection` file.